Description: Correct style errors in transmission-remote-gtk.appdata.xml
Author: Benjamin Barenblat <bbaren@debian.org>
Bug: https://github.com/transmission-remote-gtk/transmission-remote-gtk/issues/37
Forwarded: no
--- a/data/transmission-remote-gtk.appdata.xml.in
+++ b/data/transmission-remote-gtk.appdata.xml.in
@@ -14,14 +14,10 @@
   <description>
     <_p>
     Transmission Remote Gtk allows you to remotely manage the Transmission BitTorrent client using its RPC interface.
+    It is a full featured client, allowing adding, starting, stopping, removing, verifying, and reannouncing torrents.
+    It also exposes settings for both individual torrents (peer limits, file priorities, tracker settings, etc.) and Transmission as a whole (global limits, download directory, connectivity preferences, etc.)
+    It can also act as a .torrent handler, opening when torrents are downloaded through a web browser.
     </_p>
-    <_p>Features:</_p>
-    <ul>
-      <_li>Remotely add (file/url), start, stop, remove, remove and delete, verify, reannounce torrents</_li>
-      <_li>Works as a .torrent handler (eg. from a web browser)</_li>
-      <_li>Set torrent properties such as speed, seed, peer limits, file priorities, add/edit/remove trackers</_li>
-      <_li>Change remote settings like global limits, download directory, and connectivity preferences</_li>
-    </ul>
   </description>
   <screenshots>
     <screenshot height="576" width="1024" type="default">
